meetings/working-groups/extensions/Extension-API-docs.md

```diff
@@ -84,6 +85,19 @@ extension<T>(IEnumerable<T>)
 }
 ```
 
+> Alternatives: The prototypes above show a single extension member in an extension container. In source, multiple members share the same receiver and are declared in the same extension container. An alternative for the prototypes would be as follows:
+>
+> ```csharp
+> extension<T>(IEnumerable<T> source)
+> {
+>     public bool IsEmpty { get; }
+>     public int Length { get; }
+>     public IEnumerable<T> Sample(int sampleInterval);
+> }
+> ```
+>
+> It could save screen space to collect members by receiver declaration and remove the duplication. That would only apply on pages where multiple prototypes are shown together. Another negative is that it could conflict with the current lists in the TOC that have all overloads collected together.
```

Comment:
If you have multiple extension members on a single page, what would the URL of the docs page be? Something like /dotnet/api/namespace.type.extensions?

Reply:
Adding a note to define this:

```suggestion
> It could save screen space to collect members by receiver declaration and remove the duplication. That would only apply on pages where multiple prototypes are shown together. Another negative is that it could conflict with the current lists in the TOC that have all overloads collected together. We should specify the URL for the docs page for a single `extension` declaration.
```